_config.yml

```diff
@@ -2,7 +2,7 @@
 
 title: Bitcoin Core
 url: https://bitcoincore.org
-transifex_url: https://www.transifex.com/bitcoincore/bitcoincoreorg
```

Comment:
I was wondering whether this was replaced mistakenly, but it is also 404, and while there is a `https://explore.transifex.com/bitcoincore/` org on Transifex, I could not find a "bitcoincoreorg" project in it.